### PR TITLE
test: translations etc

### PR DESCRIPTION
**Pull request by haxxmaxx**

### CODE REVIEW

**Comment by haxxmaxx** [comment]
@niekvanstaveren hehe not done yet 🙃

File: src/__tests__/default-orgchart-props.js [hunk 1]
```diff
@@ -21,14 +21,17 @@ const defaultValues = {
       isLocked: false,
     },
     parent: {
-      x: 0,
-      y: 0,
+      x: 1,
+      y: 1,
+      xActual: 1,
```
Comment (haxxmaxx) [inline src/__tests__/default-orgchart-props.js]: bc 0 is falsey...

File: src/tree/render.js [hunk 2]
```diff
@@ -113,20 +113,11 @@ export const paintTree = ({
   }
 };
 
-export const getSize = ({ error, warn }, element) => {
```
Comment (haxxmaxx) [inline src/tree/render.js]: Don't thinkt his matters anymore?

File: src/tree/transform.js [hunk 3]
```diff
@@ -149,15 +149,13 @@ export function setZooming({ objectData, setTransform, transformState, selection
...
-  const { svg, divBox } = objectData;
+export const getSnapshotZoom = (rect, viewState) => {
   const { size } = viewState;
   const snapZoom = rect.width / size.w > rect.height / size.h ? rect.height / size.h : rect.width / size.w;
   const newX = viewState.transform.x * snapZoom;
   const newY = viewState.transform.y * snapZoom;
   const newZoom = viewState.transform.zoom * snapZoom;
-  const zoomEvent = zoomIdentity.translate(newX, newY).scale(newZoom);
-  applyTransform(zoomEvent, svg, divBox, rect.width, rect.height);
+  return zoomIdentity.translate(newX, newY).scale(newZoom);
```
Comment (niekvanstaveren) [inline src/tree/transform.js]: Yep good call

**Pull request merged**